src/containers/auth/EmailPasswordSignUp.jsx

```diff
@@ -0,0 +1,134 @@
+import React from "react";
+import { Form, Button, Col, Container } from "react-bootstrap";
+import { Redirect } from "react-router-dom";
+import { connect } from "react-redux";
+import { signUp } from "../../actions/authActions";
+
+const EmailPasswordSignUp = (props) => {
+  const [firstName, setFirstName] = React.useState("");
+  const [lastName, setLastName] = React.useState("");
+  const [email, setEmail] = React.useState("");
+  const [password, setPassword] = React.useState("");
+  const [passwordRepeat, setPasswordRepeat] = React.useState("");
+  const [passwordMatch, setPasswordMatch] = React.useState(true);
```

Comment:
lets call this `doesPasswordMatch, setDoesPasswordMatch` to make it clear this is a boolean